app-service-template/functions/sample.go

```diff
@@ -96,6 +97,61 @@ func (s *Sample) LogEventDetails(ctx interfaces.AppFunctionContext, data interfa
 	return true, event
 }
 
+// SendCommand is example of how to use the CommandClient to send commands to devices
+func (s *Sample) SendCommand(ctx interfaces.AppFunctionContext, data interface{}) (bool, interface{}) {
```

Comment:
since this example only demonstrates the Get command, we should rename the function to SendGetCommand

Reply:
Sure

Reply:
Renamed